pages/project/[slug].js

```diff
@@ -0,0 +1,33 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { projects } from "@/lib/db.js";
+import styled from "styled-components";
+
+export default function Tasks() {
+  const router = useRouter();
+  const { slug } = router.query;
+
+  const projectIndex = projects.findIndex((project) => project.slug === slug);
+
+  const project = projects[projectIndex];
+
+  if (!project) {
+    return null;
+  }
+
+  const { title, tasks } = project;
+
+  return (
+    <>
+      <button>
+        <Link href="/">⬅️</Link>
+      </button>
+      <h1>{title}</h1>
+      <ul>
+        {tasks.map(({ task }) => (
+          <li key={task}>{task}</li>
```

Comment:
The task might not be unique. You can add an id to your tasks though since they are already objects!